addons/website/views/website_templates.xml

```diff
@@ -304,7 +304,7 @@
         <t t-set="_navbar_expand_class" t-valuef="None"/>
         <t t-set="_navbar_name" t-valuef="Mobile"/>
 
-        <div id="top_menu_container" class="container flex-wrap justify-content-between">
+        <div id="o_main_nav" class="container flex-wrap justify-content-between">
```

Comment:
Renaming an ID is generally a bad idea for migration... but I guess that this work will generate many migration issues anyway so if there is a time to rename this ID, now is the time I guess. Let's keep it for now.